### ISSUE TITLE
Restore Apache License 2.0

### ISSUE THREAD

**Issue by yjbanov** [seaneagan/fake_async#9]
Hi, Sean,

This package is a derivative of `quiver-dart`. Quiver uses Apache License 2.0, which requires that derivatives use the same license and include it with the source code.

Please, kindly restore the LICENSE file to Apache License 2.0. All you need is copy https://github.com/google/quiver-dart/blob/master/LICENSE over https://github.com/seaneagan/fake_async/blob/master/LICENSE. In file headers, simply add your own copyright _in addition_ to the existing "Google Inc." copyright notice, like this:

```
// Copyright 2014 Google Inc. All Rights Reserved.
// Copyright 2015 fake_async authors. All Rights Reserved.
```

Cheers,
Yegor

**Comment by seaneagan** [comment]
@yjbanov You're correct of course, legally this is a derivative work.  I'll take care of it.

The reason for the choice of license was to be compatible with [other google-owned repos](http://github.com/dart-lang), so that it could be [used by them](https://code.google.com/p/dart/source/detail?r=38549). Quiver was too unwieldy for it to be pulled into the main dart repo (and other repos I needed this in) as a dependency, that's why I moved this functionality that I authored to its own package in the first place, and offered to remove it from quiver before it was released to pub.

**Comment by cbracken** [comment]
Any update on this? Apache 2.0 isn't an issue for use by the SDK. We [use Quiver in the SDK](https://github.com/dart-lang/sdk/blob/master/DEPS#L280) as-is.
